extensions/positron-catalog-explorer/eslint.config.mjs

```diff
@@ -0,0 +1,44 @@
+import typescriptEslint from "@typescript-eslint/eslint-plugin";
+import tsParser from "@typescript-eslint/parser";
```

Comment:
this should probably be removed so we get consistent eslint rules inherited from the top level

Reply:
removed!